UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs

```diff
@@ -975,12 +975,12 @@ public void UpdateLocalAndWorldConversion()
 	public void UpdateSyncVars()
 	{
 		if (isServer == false) return;
-		if (SynchronisedSpin != CurrentTorque)
+		if (Mathf.Approximately(SynchronisedSpin, CurrentTorque) == false)
```

Comment:
well isn't this suggestion from if you do stuff like 
X = 3.3333333 + 2.666666 
Y = 6
if (X == Y) 
but X Always get set as Y so there is never any scenario where it will mismatch slightly, and fail the direct comparison.
like so 
var Old = 0;
X = 3.3333333 + 2.666666 
if (X != Old ){
Old  = X
} 
(old == 5.999999)

Reply:
reverted